src/platform/intel/cavs/platform.c

```diff
@@ -270,6 +270,10 @@ static void platform_init_hw(void)
 		LPGPDMA_CHOSEL_FLAG | LPGPDMA_CTLOSEL_FLAG);
 	io_reg_write(DSP_INIT_LPGPDMA(1),
 		LPGPDMA_CHOSEL_FLAG | LPGPDMA_CTLOSEL_FLAG);
+#if defined(CONFIG_DMIC) && defined(CONFIG_ICELAKE)
+	/* Enable DMIC power */
+	io_reg_write(DMICLCTL_REG, (io_reg_read(DMICLCTL_REG) | DMIC_POWER_ON));
```

Comment:
Should we enable the power in DMIC start trigger instead to save power? And un-power in stop trigger. The power in sleep (power on, 0 Hz clock) is about 0.1 mW per microphone.

Reply:
@singalsu You are right, powering on while trigger start would be better. Also shouldn't 300ms default ramp up time (as for now) ease issue with 35-50ms lower quality issue on start?

Reply:
@singalsu, I think it is a good point. However are you sure that there will not be any other mic usage beside period DMIC start/stop trigger?